### ISSUE TITLE
We do not need to split the dataset into batches? 

### ISSUE THREAD

**Issue by BierOne** [chengsen/PyTorch_TextGCN#3]
Hi, chengsen.

I noticed that you directly train the model using the whole corpus, and you even do not split the corpus into batches. I am confused about this. Is this the way that the paper authors applied? 

Besides, if so, how can we build the connection between the input document and the target label. I cannot find any clues if we train the model in this way.

Look forward to your reply. Thx.

regards,
bill

**Comment by cdqncn** [comment]
GCN就是full batch方式
